app/components/VerificationStatusCard/VerificationStatusCard.tsx

```diff
@@ -80,7 +81,18 @@ export default function VerificationStatusCard({ credential, verifyPayload }: Ve
         <StatusItem
           positiveText="Has not been revoked by issuer"
           negativeText="Has been revoked by issuer"
-          verified={details[LogId.RevocationStatus]}
+          verified={
+            details[LogId.RevocationStatus] === undefined ||
+            details[LogId.RevocationStatus]
+          }
+        />
+        <StatusItem
```

Comment:
Let's display this item conditionally -- only when there's a `credentialStatus` section. (Ideally, we'd only display it if the bitstring status credential had suspension type lists)

Reply:
@dmitrizagidulin I have changed this to display this section only if suspension is being tracked in the credential. For consistency, I am also only displaying revocation status if it is being tracked in the credential. Let me know what you think.

Reply:
Thanks!